api/src/main/java/org/openmrs/api/OrderService.java

```diff
@@ -100,6 +102,15 @@
 	public Order saveOrder(Order order, OrderContext orderContext) throws APIException;
 
 	/**
+	 * @since 1.12
+	 * It saves a set of  orders into the database.
```

Comment:
I think this needs to have more javadocs describing exactly how this method is implemented i.e the fact that it will save an order group if necessary and assign sequence numbers to the orders based on those of the order set members in the order set from which the group was created.